### PR TITLE
For The Ember Times No. 115 - Writeup on Ember Elevator Pitch

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-09-13-the-ember-times-issue-115.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 Meet `@model` for route templates 🆕,
 Share your thoughts on Apple Music being an Ember app 🍎,
+A designer's & developer's story transitioning to Ember ⚛️➡️🐹,
```
Comment (ijlee2) [inline source/2019-09-13-the-ember-times-issue-115.md]:
```suggestion
A designer's & developer's story of transitioning to Ember ⚛️➡️🐹,
```

File: source/2019-09-13-the-ember-times-issue-115.md [hunk 2]
```diff
@@ -47,73 +48,75 @@ You can [try out `@model` now on canary](https://github.com/emberjs/ember.js/pul
...
 [Share your thoughts](https://discuss.emberjs.com/t/apple-music-beta-is-an-ember-app/17026) on having more high profile examples of Ember applications!
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [What Makes the Transition to Ember Worthwhile: A New Community Member's Perspective 🐹](https://twitter.com/lenoraporter_/status/1170145563549061122)
+
+In a recent [tweet](https://twitter.com/lenoraporter_/status/1170145563549061122), Senior Product Designer [Lenora Porter (@lenoraporter_)](https://twitter.com/lenoraporter_)
+shared a [video](https://www.instagram.com/tv/B2CCM60AOZG/?igshid=12ivaiobj1y8h) pitching the **most interesting learnings** about Ember for those transitioning from other tech stacks.
```
Comment (ijlee2) [inline source/2019-09-13-the-ember-times-issue-115.md]:
```suggestion
shared a [video](https://www.instagram.com/tv/B2CCM60AOZG/?igshid=12ivaiobj1y8h) to pitch the **most interesting learnings** about Ember to those transitioning from other tech stacks.
```

File: source/2019-09-13-the-ember-times-issue-115.md [hunk 3]
```diff
@@ -47,73 +48,75 @@ You can [try out `@model` now on canary](https://github.com/emberjs/ember.js/pul
...
+
+After developing apps using Ember with previous experience in React, she talks about learning resources that are useful for those starting out and what makes the transition worthwhile. Among benefits such as Convention over Configuration and integrated tooling, she also highlights how a supportive community makes learning Ember very straightforward.
```
Comment (ijlee2) [inline source/2019-09-13-the-ember-times-issue-115.md]: Jessica, while I understood what you mean to say, I think a more clear description may be:
```suggestion
Lenora, who has prior experience in React, talks about useful learning resources for those starting out Ember and what makes the transition worthwhile. In addition to Convention over Configuration and integrated tooling, she highlights how a supportive community makes learning Ember very straightforward.
```

**Review by ijlee2 — APPROVED** [review]
Looks good, I thought you captured Lenora's spirit well!

**Comment by jayjayjpg** [comment]
@ijlee2 Thank you for the review and all the useful wording suggestions :sparkles:

**Pull request merged**